V2/tezex/src/components/removeLiquidity/style.js

```diff
@@ -0,0 +1,337 @@
+
```

Comment:
should be a .ts rather than .js like the rest

Reply:
acknowledged